geoparquet/geoparquet-example.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>I had to install a few things to get this to work, so do you think we should add some version of the following lines?</p><blockquote>You will need to install the python library <a href="https://arrow.apache.org/docs/python/install.html" target="_blank">pyarrow</a> and the <a href="https://gdal.org/drivers/vector/parquet.html" target="_blank">parquet driver for gdal</a> for the following notebook.</blockquote><blockquote>For example</blockquote><pre class="ql-syntax" spellcheck="false">!pip install pyarrow
!mamba install -c conda-forge libgdal-arrow-parquet
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>